extensions/amp-dailymotion/0.1/test/test-amp-dailymotion.js

```diff
@@ -89,7 +89,7 @@ describes.realWin(
       const iframe = dailymotion.querySelector('iframe');
       expect(iframe).to.not.be.null;
       expect(iframe.src).to.equal(
-        'https://www.dailymotion.com/embed/video/x2m8jpp?api=1&html=1&app=amp&mute=1'
+        'https://www.dailymotion.com/embed/video/x2m8jpp?api=1&html=1&app=amp&mute=true'
```

Comment:
Should this be `1` for consistency?

Reply:
I changed it to true because we are now directly passing the result of a boolean statement into the URL.

Reply:
Ah! I saw your suggestion for how to fix that! Applying now :)